lib/src/_shared/WithUtils.jsx

```diff
@@ -1,22 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { MuiPickersContextConsumer } from '../utils/MuiPickersUtilsProvider';
 
 const withUtils = () => (Component) => {
-  const WithUtils = ({ pickerRef, ...other }, context) => {
-    if (!context.muiPickersDateUtils) {
-      // eslint-disable-next-line no-console
-      console.error('Utils should be provided');
-    }
-
-    return <Component ref={pickerRef} utils={context.muiPickersDateUtils} {...other} />;
-  };
+  const WithUtils = ({ pickerRef, ...other }) => (
```

Comment:
You can [forward ref here](https://reactjs.org/docs/context.html#forwarding-refs-to-context-consumers) better do it now  that we are still in RC.